front/app/containers/IdeasShowPage/index.tsx

```diff
@@ -55,6 +55,9 @@ const StyledIdeasShow = styled(IdeasShow)<{ hasNextPreviousControl?: boolean }>`
   `}
 
   ${media.phone`
+    ${({ theme }) => media.tablet`
+      margin-top: ${theme.menuHeight}px;
+    `}
     padding-top: ${(props) => (props.hasNextPreviousControl ? '0px' : '25px')};
```

Comment:
Not quite sure the cause of this issue/when it was introduced :thinking: But this seems to fix it at least :)